### DISCUSSION TITLE
Superscript characters disappearing

### DISCUSSION THREAD

**Discussion by Chessray** [LibrePDF/OpenPDF#764] [Q&A]
Hi there, I'm playing with OpenPDF on a small side project and am encountering issues with certain superscript characters. They just somehow get suppressed in the output, although I know from debugging that the original String object looks like it's supposed to. Can anyone help? [Issue 2](https://github.com/Chessray/math-worksheet-generator/issues/2)

**Comment by asturio** [comment]
It's quite a long time ago, but where are 2 different ways to solve this:

1. You are using Unicode characters to create the superscript, like in these Strings: "1024²" or "2 ³". In this case the superscript are special Unicoda characters, and you need to use a font, that support these. The "default" fonts don't do this. An probably you need to embed the font, so the PDF will be displayed correctly everywhere.
2. You write the "superscript" as they are, using a normal font, but smaller and placing them in the right position. Take a look at https://github.com/LibrePDF/OpenPDF/blob/master/pdf-toolbox/src/test/java/com/lowagie/examples/objects/chunk/SubSupScript.java , how this can be done.

Sad, that our user community didn't respond you earlier.
